contents.json

```diff
@@ -16439,6 +16439,33 @@
       ],
       "date_added": "May 26 2018",
       "suggested_by": "@vpeschenkov"
-    }
-  ]
+    },
+    {
+      "title": "FityIt - The shapes game",
+      "description": "FityIt is a challenging game of skill and reflexes. Not only does this game test your reaction time, is super fun and addictive. How long can you survive in FityIt?",
+      "category-ids": [
+         "game",
+         "spritekit"
+      ],
+      "source": "https://github.com/txaiwieser/FityIt",
+      "itunes": "https://itunes.apple.com/us/app/fityit-the-shapes-game/id991444581",
```

Comment:
Could you remove `us` from the iTunes link?

Reply:
Yes